scripts/tools-versions-minimum.txt

```diff
@@ -6,4 +6,4 @@ dtc=1.4.6
 gnuarmemb=9-2019-q4-major
 gnuarmemb-short=9-2019q4
 ses=5.30a
-west=0.7.0
+west=0.9.0
```

Comment:
Nothing wrong on this line, but I cannot place a comment on line 2:
https://github.com/nrfconnect/sdk-nrf/blob/4680001fd63b066eb688af3909bc31bbde99d7cf/scripts/tools-versions-minimum.txt#L2

@oyvindronningstad I believe TF-M requires that we update to a newer toolchain.

Also the use of `srec_cat` must be added.
@oyvindronningstad please provide details on this.

Reply:
Yes. TFM requires Cmake 3.15.0 or above.
srec_cat is needed for TFM, yes, but we can actually replace it with a 30-line python script, so I want to take a look at that.

Reply:
ok, I added: 

scripts/tools-versions-minimum.txt
-cmake=3.13.1
+cmake=3.15.0